BlueElectrum.js

```diff
@@ -20,12 +19,29 @@ const hardcodedPeers = [
 let mainClient = false;
 let mainConnected = false;
 let wasConnectedAtLeastOnce = false;
+let currentHost = '';
+
+const getHost = () => {
+  const hosts = config.hosts;
+  const hostsLength = hosts.length;
```

Comment:
I'm not convinced if having such an extra variable has sense. It doesn't hurt, but while reading you must check what is `hostsLength`. I think in the case of `hosts.length` it is more readable, but I'm aware it's very subjective.